crates/matrix-sdk-crypto/src/olm/group_sessions/inbound.rs

```diff
@@ -832,13 +834,15 @@ mod tests {
 
         // And we populated the InboundGroupSession's sender_data with the provided
         // values
-        let SenderData::UnknownDevice { retry_details, legacy_session } = unpickled.sender_data
+        let SenderData::UnknownDevice { retry_details, legacy_session, owner_check_failed } =
+            unpickled.sender_data
         else {
             panic!("Expected sender_data to be UnknownDevice!");
         };
```

Comment:
This one could have used `assert_let` as well.

Reply:
Thanks, no idea what happened to my brain here :-) Fixed in f05f624521